### ISSUE TITLE
Unable to create Serverless API using SAM v1.40.0

### ISSUE THREAD

**Issue by sdsa-cci** [aws/aws-sam-cli#3692]
### Description:

I tried creating a SAM application using the quick start template for serverless API, but unfortunately it complains that it's 
unable to local the quickstart-web template.

### Steps to reproduce:

```
D:\Projects
λ sam init

You can preselect a particular runtime or package type when using the `sam init` experience.
Call `sam init --help` to learn more.

Which template source would you like to use?
        1 - AWS Quick Start Templates
        2 - Custom Template Location
Choice: 1

Choose an AWS Quick Start application template
        1 - Hello World Example
        2 - Multi-step workflow
        3 - Serverless API
        4 - Scheduled task
        5 - Standalone function
        6 - Data processing
        7 - Infrastructure event management
        8 - Machine Learning
Template: 3

Which runtime would you like to use?
        1 - dotnetcore3.1
        2 - nodejs14.x
        3 - nodejs12.x
        4 - python3.9
        5 - python3.8
Runtime: 2

Based on your selections, the only Package type available is Zip.
We will proceed to selecting the Package type as Zip.

Based on your selections, the only dependency manager available is npm.
We will proceed copying the template using npm.

Project name [sam-app]: sammy

Cloning from https://github.com/aws/aws-sam-cli-app-templates (process may take a moment)
Error: Can't find application template quick-start-web - check valid values in interactive init.
```

### Observed result:

**Error: Can't find application template quick-start-web - check valid values in interactive init.**

### Expected result:

I expected it to create Serverless API project with it's default scaffolding. 

### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)

1. OS: Windows 10 Pro
2. `sam --version`: SAM CLI, version 1.40.0
3. AWS region: us-east-2

**Comment by mingkun2020** [comment]
I tried the same `sam init` flow above and it worked on my local SAM CLI, version 1.40.0. Could you provide more detail with `--debug` option?

**Comment by sdsa-cci** [comment]
@mingkun2020 Here's the output with the `--debug` flag. 

```
D:\Projects
λ sam init --debug
2022-02-27 18:04:29,101 | Telemetry endpoint configured to be https://aws-serverless-tools-telemetry.us-west-2.amazonaws.com/metrics
2022-02-27 18:04:29,102 | Using config file: samconfig.toml, config environment: default
2022-02-27 18:04:29,103 | Expand command line arguments to:
2022-02-27 18:04:29,103 | --package_type=Zip --output_dir=.

You can preselect a particular runtime or package type when using the `sam init` experience.
Call `sam init --help` to learn more.

Which template source would you like to use?
        1 - AWS Quick Start Templates
        2 - Custom Template Location
Choice: 1

Choose an AWS Quick Start application template
        1 - Hello World Example
        2 - Multi-step workflow
        3 - Serverless API
        4 - Scheduled task
        5 - Standalone function
        6 - Data processing
        7 - Infrastructure event management
        8 - Machine Learning
Template: 3

Which runtime would you like to use?
        1 - dotnetcore3.1
        2 - nodejs14.x
        3 - nodejs12.x
        4 - python3.9
        5 - python3.8
Runtime: 2

Based on your selections, the only Package type available is Zip.
We will proceed to selecting the Package type as Zip.

Based on your selections, the only dependency manager available is npm.
We will proceed copying the template using npm.

Project name [sam-app]: sammy
2022-02-27 18:04:42,602 |
Cloning from https://github.com/aws/aws-sam-cli-app-templates (process may take a moment)
2022-02-27 18:04:48,610 | Clone error, attempting to use an old clone from a previous run
2022-02-27 18:04:48,610 | Template is not provided in context, skip adding project type metric
2022-02-27 18:04:48,611 | Sending Telemetry: {'metrics': [{'commandRun': {'requestId': '5cd212a8-53b4-41aa-87f5-9917f6e61fc7', 'installationId': '0f72fa30-600b-4ac6-abb6-6e9191ddd5a8', 'sessionId': 'e4aa96cc-73c2-4f1e-84d4-8b0bdfb26231', 'executionEnvironment': 'CLI', 'ci': False, 'pyversion': '3.8.8', 'samcliVersion': '1.40.0', 'awsProfileProvided': False, 'debugFlagProvided': True, 'region': '', 'commandName': 'sam init', 'duration': 19510, 'exitReason': 'InvalidInitTemplateError', 'exitCode': 1}}]}
2022-02-27 18:04:49,558 | HTTPSConnectionPool(host='aws-serverless-tools-telemetry.us-west-2.amazonaws.com', port=443): Read timed out. (read timeout=0.1)
Error: Can't find application template quick-start-web - check valid values in interactive init.
```

**Comment by jfuss** [comment]
@sdsa-cci It looks like cloning the repo failed on your machine `Clone error, attempting to use an old clone from a previous run`. This explains the difference but not why this fails. 

That error gets thrown from https://github.com/aws/aws-sam-cli/blob/bad9a6226d0b871bf531ba649ee12a6ca90aa984/samcli/commands/init/init_templates.py#L71 but is executed from our clone throwing an error: https://github.com/aws/aws-sam-cli/blob/bad9a6226d0b871bf531ba649ee12a6ca90aa984/samcli/lib/utils/git_repo.py#L84. Not entirely sure what the cause is here but things I can think of to check:
1) Do you have git installed?
2) Are you able to clone https://github.com/aws/aws-sam-cli-app-templates (this is the repo for our templates)?
3) You could try to run a git clone into a temp directory (what we do here: https://github.com/aws/aws-sam-cli/blob/bad9a6226d0b871bf531ba649ee12a6ca90aa984/samcli/lib/utils/git_repo.py#L127)? Maybe its something to do with that?

**Comment by sdsa-cci** [comment]
@jfuss Yes, I do have git installed and I'm able to clone the repo successfully, but only the `sam init` command seems to be failing. 

```
λ git --version
git version 2.35.1.windows.2

D:\Projects
λ git clone https://github.com/aws/aws-sam-cli-app-templates.git
Cloning into 'aws-sam-cli-app-templates'...
remote: Enumerating objects: 4839, done.
remote: Counting objects: 100% (1326/1326), done.
remote: Compressing objects: 100% (320/320), done.
remote: Total 4839 (delta 1162), reused 1044 (delta 1003), pack-reused 3513Receiving objectsReceiving objects: 100% (4839/4839), 20.97 MiB | 9.02 MiB/s, done.
Resolving deltas: 100% (3230/3230), done.

D:\Projects
λ cd aws-sam-cli-app-templates\

D:\Projects\aws-sam-cli-app-templates(master -> origin)
λ ls -lt
total 149
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 tests/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 ruby2.7/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 ruby2.7-image/
-rw-r--r-- 1 salvino.dsa 1049089    85 Mar  2 09:46 requirements.txt
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 python3.9/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 python3.9-image/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 python3.8/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 python3.8-image/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 python3.7/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 python3.7-image/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 python3.6/
-rw-r--r-- 1 salvino.dsa 1049089   112 Mar  2 09:46 pyproject.toml
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 python3.6-image/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 nodejs14.x/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 nodejs14.x-image/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 nodejs12.x/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 nodejs12.x-image/
-rw-r--r-- 1 salvino.dsa 1049089 40186 Mar  2 09:46 manifest.json
-rw-r--r-- 1 salvino.dsa 1049089 31769 Mar  2 09:46 manifest-v2.json
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 java8/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 java8.al2/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 java8.al2-image/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 java8-image/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 java11/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 java11-image/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 go1.x/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 go1.x-image/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 dotnetcore3.1/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 dotnetcore3.1-image/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 dotnet6/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 dotnet6-image/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 buildspecs/
drwxr-xr-x 1 salvino.dsa 1049089     0 Mar  2 09:46 dotnet5.0-image/
-rw-r--r-- 1 salvino.dsa 1049089     0 Mar  2 09:46 __init__.py
-rw-r--r-- 1 salvino.dsa 1049089  2947 Mar  2 09:46 buildspec.yml
-rw-r--r-- 1 salvino.dsa 1049089   321 Mar  2 09:46 README.md
-rw-r--r-- 1 salvino.dsa 1049089 10317 Mar  2 09:46 LICENSE
-rw-r--r-- 1 salvino.dsa 1049089    73 Mar  2 09:46 NOTICE
-rw-r--r-- 1 salvino.dsa 1049089   313 Mar  2 09:46 CODE_OF_CONDUCT.md
-rw-r--r-- 1 salvino.dsa 1049089  3367 Mar  2 09:46 CONTRIBUTING.md
```

**Comment by jfuss** [comment]
@sdsa-cci Interesting. Not sure why this would happen. I guess did manually clone unblock what you are trying to do? Not a long term solution but at least makes it usable again.

I am trying to think why this could happen. I wonder if it relates to different drives. Looks like where we clone is on `C` but you are on `D`.

**Comment by sdsa-cci** [comment]
@jfuss yeah, with manual clone I'm able to clone the SAM templates, but it doesn't really improve my situation. It clones the projects with cookiecutter directories and names. Off course I could rename & remove the unwanted `cookiecutter.json` but then again that's won't help with in long term. 

Additionally, I also tried hitting `sam init` in the `C` drive, but it's the same scenario. 
Attaching the output below:
```

C:\test                                                                                                                                                                                                                                       
λ sam init --debug                                                                                                                                                                                                                            
2022-03-08 12:22:06,189 | Telemetry endpoint configured to be https://aws-serverless-tools-telemetry.us-west-2.amazonaws.com/metrics                                                                                                          
2022-03-08 12:22:06,189 | Using config file: samconfig.toml, config environment: default                                                                                                                                                      
2022-03-08 12:22:06,205 | Expand command line arguments to:                                                                                                                                                                                   
2022-03-08 12:22:06,205 | --package_type=Zip --output_dir=.                                                                                                                                                                                   
                                                                                                                                                                                                                                              
You can preselect a particular runtime or package type when using the `sam init` experience.                                                                                                                                                  
Call `sam init --help` to learn more.                                                                                                                                                                                                         
                                                                                                                                                                                                                                              
Which template source would you like to use?                                                                                                                                                                                                  
        1 - AWS Quick Start Templates                                                                                                                                                                                                         
        2 - Custom Template Location                                                                                                                                                                                                          
Choice: 1                                                                                                                                                                                                                                     
                                                                                                                                                                                                                                              
Choose an AWS Quick Start application template                                                                                                                                                                                                
        1 - Hello World Example                                                                                                                                                                                                               
        2 - Multi-step workflow                                                                                                                                                                                                               
        3 - Serverless API                                                                                                                                                                                                                    
        4 - Scheduled task                                                                                                                                                                                                                    
        5 - Standalone function                                                                                                                                                                                                               
        6 - Data processing                                                                                                                                                                                                                   
        7 - Infrastructure event management                                                                                                                                                                                                   
        8 - Machine Learning                                                                                                                                                                                                                  
Template: 3                                                                                                                                                                                                                                   
                                                                                                                                                                                                                                              
Which runtime would you like to use?                                                                                                                                                                                                          
        1 - dotnetcore3.1                                                                                                                                                                                                                     
        2 - nodejs14.x                                                                                                                                                                                                                        
        3 - nodejs12.x                                                                                                                                                                                                                        
        4 - python3.9                                                                                                                                                                                                                         
        5 - python3.8                                                                                                                                                                                                                         
Runtime: 2                                                                                                                                                                                                                                    
                                                                                                                                                                                                                                              
Based on your selections, the only Package type available is Zip.                                                                                                                                                                             
We will proceed to selecting the Package type as Zip.                                                                                                                                                                                         
                                                                                                                                                                                                                                              
Based on your selections, the only dependency manager available is npm.                                                                                                                                                                       
We will proceed copying the template using npm.                                                                                                                                                                                               
                                                                                                                                                                                                                                              
Project name [sam-app]:                                                                                                                                                                                                                       
2022-03-08 12:22:30,679 |                                                                                                                                                                                                                     
Cloning from https://github.com/aws/aws-sam-cli-app-templates (process may take a moment)                                                                                                                                                     
2022-03-08 12:22:35,962 | Clone error, attempting to use an old clone from a previous run                                                                                                                                                     
2022-03-08 12:22:35,965 | Template is not provided in context, skip adding project type metric                                                                                                                                                
2022-03-08 12:22:35,965 | Sending Telemetry: {'metrics': [{'commandRun': {'requestId': 'ae3c316e-e383-440d-ab43-be412fb5860e', 'installationId': '0f72fa30-600b-4ac6-abb6-6e9191ddd5a8', 'sessionId': '70c6bd46-f132-45ea-bac5-21eb6ee46c64', 
'executionEnvironment': 'CLI', 'ci': False, 'pyversion': '3.8.8', 'samcliVersion': '1.40.0', 'awsProfileProvided': False, 'debugFlagProvided': True, 'region': '', 'commandName': 'sam init', 'duration': 29762, 'exitReason': 'InvalidInitTem
plateError', 'exitCode': 1}}]}                                                                                                                                                                                                                
2022-03-08 12:22:36,933 | HTTPSConnectionPool(host='aws-serverless-tools-telemetry.us-west-2.amazonaws.com', port=443): Read timed out. (read timeout=0.1)                                                                                    
Error: Can't find application template quick-start-web - check valid values in interactive init.      
```

**Comment by kamsar** [comment]
I ran into this issue as well. In my case:

* Only one drive (C:\)
* I had a much older clone of the repository sitting around at `%APPDATA%\AWS SAM\aws-sam-cli-app-templates`
* As with @sdsa-cci, the `--debug` flag gives the generic `Clone error` message with no further details
* No HTTP proxies or anything going on - I do have a DNS-layer adblock, possibly from a blocked telemetry call?
* The user I ran `sam` as has write access to the `%APPDATA%\AWS SAM` folder

I was able to fix it by:

1. Open `%APPDATA%\AWS SAM`
2. Delete the `aws-sam-cli-app-templates` folder
3. Run `git clone https://github.com/aws/aws-sam-cli-app-templates.git` to re-populate it

Obviously this does not solve the clone error problem, but it does unblock usage of `sam init`.

**Comment by sdsa-cci** [comment]
Hi @kamsar really appreciate your response. Unfortunately, I had a look in the `%APPDATA%\AWS SAM` directory, but could not find any `aws-sam-cli-app-templates` folder in it.

**Comment by kamsar** [comment]
Could try cloning it fresh there with git, might work if it had the right name

**Comment by sdsa-cci** [comment]
@kamsar Thank you so much! Cloning the repository manually in to the app data directory solved the problem. @jfuss Maybe that's the problem. `SAM cli` has some issue cloning to app data directories and can be looked into. 

For those who stumble upon the same thing, you can do the following to solve it temporarily:
1. Open the start menu and type `run` and open the run command dialog. 
2.  Type `%APPDATA%\AWS SAM` and click `OK`.
3. Look for a folder called `aws-sam-cli-app-templates`
4. If the folder exists, delete it or if it doesn't follow the next step. 
5. Open `cmd` in the current directory and then clone the sam cli templates repository manually. 
`git clone https://github.com/aws/aws-sam-cli-app-templates.git`
6. That's it! Now try running `sam init` in your actual workspace directory.

**Comment by qingchm** [comment]
@sdsa-cci Glad that the issue is now unblocked! Yes for those situations where the app templates are not functioning as expected, which fails the init command, going into the designated aws-sam-cli-app-templates location can help! I've also encountered something similar before that I had to wipe out that folder for a re-clone. Let me know if you think any changes from our side can help improve this experience! Since this issue is resolved, closing the issue for now!

**Comment by sdsa-cci** [comment]
> @sdsa-cci Glad that the issue is now unblocked! Yes for those situations where the app templates are not functioning as expected, which fails the init command, going into the designated aws-sam-cli-app-templates location can help! I've also encountered something similar before that I had to wipe out that folder for a re-clone. Let me know if you think any changes from our side can help improve this experience! Since this issue is resolved, closing the issue for now!

@qingchm I would like the team to check why the clone to `%APPDATA%\AWS SAM` fails in the first place. It would be good, it this issue can be looked into and fixed in the upcoming releases. For now I can continue to use the work around of cloning the repository manually.

**Comment by duterte** [comment]
I encounter the same scenario when choosing "Serverless API" on the second prompt. I followed the same steps and was resolved. But I think that is not an Ideal fix. I hope that maintainer cloud look what's wrong with choosing Serverless API

**Comment by alan-wint** [comment]
This was closed prematurely imo.  Latest packaged build for Windows (AWS_SAM_CLI_64_PY3.msi 2022-05-04) still has this issue.

**Comment by nicolefeatherby** [comment]
Trying to clone the repo manually as described above gives me this error:

```
PS C:\Users\Nicole\AppData\Roaming\AWS SAM> git clone https://github.com/aws/aws-sam-cli-app-templates.git
Cloning into 'aws-sam-cli-app-templates'...
remote: Enumerating objects: 5561, done.
remote: Counting objects: 100% (1660/1660), done.
remote: Compressing objects: 100% (491/491), done.
remote: Total 5561 (delta 1419), reused 1238 (delta 1164), pack-reused 3901
Receiving objects: 100% (5561/5561), 21.09 MiB | 10.57 MiB/s, done.
Resolving deltas: 100% (3709/3709), done.
fatal: cannot create directory at 'provided.al2/graalvm/java11/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/HelloWorldFunction/src/main/resources/META-INF/native-image/com.amazonaws/aws-lambda-java-core': Filename too long
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
```

**Comment by tbenbrahim** [comment]
Not sure why this is closed
```
SAM CLI, version 1.51.0
```
Project name [sam-app]:
2022-06-03 16:41:18,402 |
Cloning from https://github.com/aws/aws-sam-cli-app-templates (process may take a moment)
2022-06-03 16:41:39,662 | Clone error, attempting to use an old clone from a previous run
2022-06-03 16:41:39,662 | Template is not provided in context, skip adding project type metric
2022-06-03 16:41:39,663 | Sending Telemetry: {'metrics': [{'commandRun': {'requestId': '833fd30f-f00f-4419-8454-e861cd9f0c4c', 'installationId': '0eee692d-1eaa-4416-8590-39364ab2554d', 'sessionId': '43c27db2-eba4-4afe-ba84-b37fee27de3d', 'executionEnvironment': 'CLI', 'ci': False, 'pyversion': '3.8.8', 'samcliVersion': '1.51.0', 'awsProfileProvided': False, 'debugFlagProvided': True, 'region': '', 'commandName': 'sam init', 'duration': 34543, 'exitReason': 'InvalidInitTemplateError', 'exitCode': 1}}]}
2022-06-03 16:41:40,656 | HTTPSConnectionPool(host='aws-serverless-tools-telemetry.us-west-2.amazonaws.com', port=443): Read timed out. (read timeout=0.1)
Error: Can't find application template step-functions-sample-app - check valid values in interactive init.
```

```

**Comment by laPlacek** [comment]
same issue here. just installed sam and i get the same message
`
C:\dev>sam --version
SAM CLI, version 1.51.0
`

`
C:\dev>sam init --debug
2022-06-07 19:47:41,166 | Telemetry endpoint configured to be https://aws-serverless-tools-telemetry.us-west-2.amazonaws.com/metrics
2022-06-07 19:47:41,166 | Using config file: samconfig.toml, config environment: default
2022-06-07 19:47:41,166 | Expand command line arguments to:
2022-06-07 19:47:41,167 | --package_type=Zip --output_dir=.

You can preselect a particular runtime or package type when using the `sam init` experience.
Call `sam init --help` to learn more.

Which template source would you like to use?
        1 - AWS Quick Start Templates
        2 - Custom Template Location
Choice: 1

Choose an AWS Quick Start application template
        1 - Hello World Example
        2 - Multi-step workflow
        3 - Serverless API
        4 - Scheduled task
        5 - Standalone function
        6 - Data processing
        7 - Infrastructure event management
        8 - Machine Learning
Template: 5

Which runtime would you like to use?
        1 - dotnet6
        2 - dotnetcore3.1
        3 - nodejs16.x
        4 - nodejs14.x
        5 - nodejs12.x
Runtime: 4

Based on your selections, the only Package type available is Zip.
We will proceed to selecting the Package type as Zip.

Based on your selections, the only dependency manager available is npm.
We will proceed copying the template using npm.

Would you like to enable X-Ray tracing on the function(s) in your application?  [y/N]:

Project name [sam-app]:
2022-06-07 19:48:05,895 |
Cloning from https://github.com/aws/aws-sam-cli-app-templates (process may take a moment)
2022-06-07 19:48:09,553 | Clone error, attempting to use an old clone from a previous run
2022-06-07 19:48:09,554 | Template is not provided in context, skip adding project type metric
2022-06-07 19:48:09,554 | Sending Telemetry: {'metrics': [{'commandRun': {'requestId': 'dd27e9d4-6b4e-4cfc-82f4-bf0c304ca4d9', 'installationId': '55257039-e670-490c-a5ae-185640c6b00d', 'sessionId': '956bd29f-6429-4943-bf3b-15756717488e', 'executionEnvironment': 'CLI', 'ci': False, 'pyversion': '3.8.8', 'samcliVersion': '1.51.0', 'awsProfileProvided': False, 'debugFlagProvided': True, 'region': '', 'commandName': 'sam init', 'duration': 28387, 'exitReason': 'InvalidInitTemplateError', 'exitCode': 1}}]}
2022-06-07 19:48:10,289 | HTTPSConnectionPool(host='aws-serverless-tools-telemetry.us-west-2.amazonaws.com', port=443): Read timed out. (read timeout=0.1)
Error: Can't find application template quick-start-from-scratch - check valid values in interactive init.
`

**Comment by Exaustia** [comment]
Hello, same issue here. just installed sam 
![image](https://user-images.githubusercontent.com/10464243/173333364-cd5ca83a-d152-44dd-87c5-60db17de66d6.png)

**Comment by Exaustia** [comment]
Hello, 
Same issue here :(

**Comment by hawflau** [comment]
Looks like it's the same issue as #3946, which was caused by long path limitation in Windows. We have a fix #3950 merged but not released yet

**Comment by hawflau** [comment]
Meanwhile, please refer [here](https://github.com/aws/aws-sam-cli/issues/3946#issuecomment-1150121237) for a workaround

**Comment by mndeveci** [comment]
Related issue is been fixed and released in latest 1.53.0 version. Please give it a try and let us know if you still experience similar issues.

**Comment by raffaeu** [comment]
Just per my experience:

- Operating System **Windows 11**
- Terminal Version **GitBash for Windows**

I was receiving this error:

```
Cloning from https://github.com/aws/aws-sam-cli-app-templates (process may take a moment)
Error: Can't find application template quick-start-web - check valid values in interactive init.
```

I fixed by heading to: `%APPDATA%\AWS SAM`

Then I deleted everything and issued in that folder a `git clone git@github.com:aws/aws-sam-cli-app-templates.git`. This happens because GITHUB has changed credentials mechanism recently, so if you issue `sam init` from your local PC and you didn't update the ssh key, then you simply can't clone the repo but SAM CLI gives you a slightly different error message.
